index.ts

```diff
@@ -17,3 +15,4 @@ export { default as Tabs } from "./src/navigation/Tabs"
 export { default as Heading } from "./src/text/Heading"
 export { default as HeadingGroup } from "./src/text/HeadingGroup"
 export { default as Tag } from "./src/text/Tag"
+export { default as ExpandableText } from "./src/text/ExpandableText"
```

Comment:
I don't know if we ever vetted this as a true Seeds component…I believe it was test code at the inception of the repo. Might need a design review? (I also wonder if we can move towards semantic HTML for things like this…aka `<details>` + `<summary>`)

Reply:
Sounds good! I'll comment out the stories for now 🫡